src/core/referencingfeaturelistmodel.h

```diff
@@ -0,0 +1,96 @@
+/***************************************************************************
+  referencingfeaturelistmodel.h - ReferencingFeatureListModel
+
+ ---------------------
+ begin                : 1.3.2019
+ copyright            : (C) 2019 by David Signer
+ email                : david@opengis.ch
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+#ifndef REFERENCINGFEATURELISTMODEL_H
+#define REFERENCINGFEATURELISTMODEL_H
+
+#include <QAbstractItemModel>
+#include <QPair>
+#include "qgsvectorlayer.h"
+#include "attributeformmodel.h"
+
+class QgsVectorLayer;
+class AttributeFormModel;
+
+class ReferencingFeatureListModel : public QStandardItemModel
```

Comment:
Should this be a `QAbstractItemModel` instead? I think you don't use any of the standarditemmodel logic in here.